index.js

```diff
@@ -170,30 +177,29 @@ const recastNewUsers = async () => {
         break
       }
 
-      const { body, meta } = cast
-      const { address, data, publishedAt, sequence, username } = body
-      if (meta.recast) {
+      const { hash, timestamp, recasts, parentHash, text, author } = cast
+      if (recasts?.count) {
```

Comment:
`recasts.count` doesn't indicate that this cast was recast from the author.

we should check `cast.recast` instead

Reply:
done